compiler/grainformat/reformat.re

```diff
@@ -512,11 +539,15 @@ and resugar_list =
     ) => {
   let processed_list = resugar_list_inner(expressions, parent_loc);
 
+  let last_item_was_spread = ref(false);
+
   let items =
     List.map(
       i =>
         switch (i) {
         | Regular(e) =>
+          last_item_was_spread := false;
```

Comment:
I wish we didn't need to use a mutable variable here, but I can't think of a way to nicely clean it up without one :/

Reply:
Me too, I hate myself each time I use a mutable variable.

Reply:
🥲